source/MRMesh/MRDistanceMapLoad.h

```diff
@@ -23,7 +24,7 @@ MRMESH_API extern const IOFilters Filters;
  * 2 integer - DistanceMap.resX & DistanceMap.resY
  * [resX * resY] float - matrix of values
  */
-MRMESH_API tl::expected<DistanceMap, std::string> loadRaw( const std::filesystem::path& path, ProgressCallback progressCb = {} );
+MRMESH_API tl::expected<DistanceMapWithParams, std::string> loadRaw( const std::filesystem::path& path, ProgressCallback progressCb = {} );
```

Comment:
mb also add function like `fromAnySupportedFormat`? (and raname `loadRaw` to `fromRaw`)